h/static/scripts/directives/markdown.coffee

```diff
@@ -336,7 +336,7 @@ markdown = ['$filter', '$sanitize', '$sce', '$timeout', ($filter, $sanitize, $sc
     # React to the changes to the input
     inputEl.bind 'blur change keyup', ->
       ctrl.$setViewValue inputEl.val()
-      scope.$digest()
+#      scope.$digest()  # TODO: this shouldn't be a problem
```

Comment:
Detailed instructions to reproduce:
 - Uncomment [this line]https://github.com/hypothesis/h/blob/export-showframe/h/static/scripts/directives/markdown.coffee#L339
 - Run code
 - Select something in the document
 - Press the adder button

Thanks.